src/lib.rs

```diff
@@ -854,6 +884,8 @@ fn test_prefix() {
             ("HOME", format!("{}/test_files/user", cwd)),
             ("XDG_CACHE_HOME", format!("{}/test_files/user/cache", cwd)),
         ])).unwrap();
+    assert_eq!(xd.get_cache_file("cache.db"),
+        PathBuf::from(&format!("{}/test_files/user/cache/myapp/cache.db", cwd)));
```

Comment:
Can you add tests for the other functions too?

Reply:
Done!